pobbin/util/pob_xml_verifier.py

```diff
@@ -0,0 +1,25 @@
+from fastapi.logger import logger as fastapi_logger
+
+from lxml import etree
+
+from pobbin import load_file_as_string
+
+
+class XsdVerifier:
```

Comment:
This should just be a function instead, instead of a namespace inside a namespace. If you worry about switching out `xsd_path` for testing or polluting the signature of `is_valid_pob` with a DI, consider making a one-argument `is_valid_pob` with `functools.partial` instead. No need for (faux) singletons in Python.

Reply:
I was mainly shooting for keeping the XSD in memory but this should do the same:

```py
XSD_PATH = "assets/pob_build.xsd"
XSD_CONTENT = load_file_as_string(XSD_PATH)


def is_valid_pob(pob_build_xml: str) -> bool:
    schema_doc = etree.XML(XSD_CONTENT)
    schema = etree.XMLSchema(schema_doc)
    parser = etree.XMLParser(schema=schema)
    try:
        etree.XML(pob_build_xml, parser)
    except etree.XMLSyntaxError as e:
        logger.warning(f"Incoming build does not match DTD: '{e}'")
        return False
    return True
```

Reply:
I see. The new version communicates your intent more clearly I think.